osmdroid-shape/src/main/java/org/osmdroid/shape/ShapeConverter.java

```diff
@@ -39,7 +41,22 @@
 
 public class ShapeConverter {
 
-    public static List<Overlay> convert(MapView map, File file, ValidationPreferences prefs) throws Exception {
+    /**
+     *
+     * @param map the MapView to which these overlays will be added.
+     * @param file the shape file to be converted.
+     * @param prefs  allows the client to relax the level of validation when reading a shape file.
+     * @param fillColor fill color of polygon.
+     * @param outLineColor out line color of polyOverlay.
+     * @param width out line width of polyOverlay.
+     * @param getBubbleText customize titles, snippets, and sub-descriptions of bubbles.
+     *                      remember to handling bounding box if your shape file has both points and poly-overlays.
+     *                      please also mind that getSnippet is called before getTitle, and before getSubDescription.
+     * @return
+     * @throws Exception
+     */
+    public static List<Overlay> convert(MapView map, File file, ValidationPreferences prefs,
```

Comment:
I guess `int strokeWidth` and `int strokeColor` would make more sense.

Reply:
Thank you for the fix.

Now that I'm thinking it would make much more sense if instead of the fill color, stroke color and stroke width we used 2 nullable parameters instead : `Paint fillPaint` and `Paint strokePaint`. And use them in the code with something like
````java
if (fillPaint != null) {
    polygon.getFillPaint().set(fillPaint);
}
````


It's better because we have much more control over the polygon paint (rounding the segments ending, dashed strokes, ...). And we don't have to create another version of `convert` if we say "now I'd like to add a style to all the shape polygons".